internal/internal_task_handlers.go

```diff
@@ -824,9 +824,12 @@ process_Workflow_Loop:
 func (w *workflowExecutionContextImpl) ProcessWorkflowTask(workflowTask *workflowTask) (interface{}, error) {
 	task := workflowTask.task
 	historyIterator := workflowTask.historyIterator
+	w.workflowInfo.TotalHistoryBytes = task.GetTotalHistoryBytes()
+	w.workflowInfo.HistoryCount = task.GetNextEventId() - 1
```

Comment:
I'm not sure if we want to use this or just count locally... or at least if it's not present.

Reply:
or actually this might be a source of problems.

this is exposed in workflow info, which is (intentionally) exposed to workflow _code_, but it's non-deterministic data: at event N you'll see changing values depending on how many events have been executed in the future.

count we can probably just keep track locally, bytes... we might have to estimate based on the serialized data?